CryptoTechReminderSystem.Test/UseCase/ShameLateDevelopersTests.cs

```diff
@@ -92,5 +91,22 @@ public void CanCheckMessageHadChannel()
             
             _sendReminderSpy.Channels.First().Should().Be(expectedChannel);
         }
+
+        [Test]
+        public void CanSendNoShameMessageWhenNoLateDevelopers()
+        {
+            var getLateDevelopersEmptyStub = new GetLateDevelopersEmptyStub();
+            var shameLateDevelopers = new ShameLateDevelopers(getLateDevelopersEmptyStub, _sendReminderSpy);
+            var expectedNoShameMessage = "Everyone has submitted their timesheets!";
```

Comment:
This could probably be a constant.

```suggestion
            const string expectedNoShameMessage = "Everyone has submitted their timesheets!";
```